schema/aiconfig.schema.json

```diff
@@ -208,7 +232,79 @@
               "type": "number"
             },
             "data": {
-              "description": "The result of executing the prompt."
+              "description": "The result of executing the prompt.",
+              "anyOf": [
+                {
+                  "type": "object",
+                  "additionalProperties": {}
+                },
+                {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/definitions/JSONValue"
+                  }
+                },
```

Comment:
Weird that this doesn't just use the JSONValue ref

Reply:
Good question